presto-native-execution/presto_cpp/main/PrestoServer.cpp

```diff
@@ -656,6 +663,17 @@ void PrestoServer::run() {
   }
 }
 
+void PrestoServer::startLinuxMemoryChecker() {
+#ifdef PRESTO_ENABLE_LINUX_MEMORY_CHECKER
+  PeriodicMemoryChecker::Config config;
+  config.systemMemPushbackEnabled = systemConfig->systemMemPushbackEnabled();
```

Comment:
Do we still want setup the memory checker if the pushback is not enabled ?

Reply:
Depends. The heap dumper is independent to the memory pushback. If we are not going to implement the heap dumper then we can skip the whole setup

Reply:
Added comment 
```
// If heap dumper is to not be implemented for LinuxMemoryChecker, 
  // then this setup for LinuxMemoryChecker can be skipped.
  ```